test/test_sklearn_compat.py

```diff
@@ -72,7 +72,7 @@ def test_itml(self):
   def test_mmc(self):
     check_estimator(dMMC)
 
-  # This fails due to a FloatingPointError
+  # TODO: uncomment this when SDML has become deterministic
```

Comment:
I would have liked to uncomment this since there is not FloatingPointError anymore, but there is still the undeterminacy of SDML that makes it fail